src/js/popup/view/view.js

```diff
@@ -1,82 +0,0 @@
-/**
- * @file Contains UI of popup
- */
-
-import { isTimerActive } from "../actions/timer";
-import { updateTimerCounter } from "./timerCounterView";
-
-import imgStart from "../../../img/start-gray-16.png";
-import imgStop from "../../../img/stop-red-16.png";
-
-const timerBtn = document.getElementById("timerToggler");
-
-function updateView(data) {
-  updateButtonView(isTimerActive(data));
-  updateInputsView(data.timer.projectId, data.timer.mergeRequestId);
```

Comment:
Please fix this bug:
1. Open MR1;
2. Start tracking;
3. Go to MR2;
4. Open popup and see that MR2 is chosen but MR1 shall be.